app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
+  updateSaveAndContinueVisibility: function() {
```

Comment:
This code looks a bit over engineered. I would replace this whole method with:

```
$("#checkout_form_payment [data-hook=buttons]").toggle(this.isButtonHidden());
```
and would remove those unnecessary methods like: hideSaveAndContinue and showSaveAndContinue.